monero-wallet-generator.html

```diff
@@ -11544,6 +11545,24 @@ <h5>Checking a git commit's signature</h5>
   mnemonic_widget.style.display = "block";
 }
 
+function shouldDimoutPage(mustDimout) {
+    dimoutElmnt = document.getElementById('dimoutPage');
+    if (dimoutElmnt==undefined) return;
+
+    if (mustDimout==true) {
+        dimoutPage.style.display = 'block'
+        dimoutPage.style.width ='100%'
+        dimoutPage.style.height ='100%'
+        dimoutPage.style.position = 'fixed'
+        dimoutPage.style.top = '0'
+        dimoutPage.style.backdropFilter='blur(2px)'
+        dimoutPage.style.zIndex = '100'
```

Comment:
All this can be in the original element already. No need for dynamic changes beyond display block/none.